tests/unit/config/badge-group-names-test.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Yoast\WP\SEO\Tests\Unit\Config;
+
+use Yoast\WP\SEO\Config\Badge_Group_Names;
+use Yoast\WP\SEO\Tests\Unit\TestCase;
+
+/**
+ * Class Badge_Group_Names_Test.
+ *
+ * @coversDefaultClass \Yoast\WP\SEO\Config\Badge_Group_Names
+ */
+class Badge_Group_Names_Test extends TestCase {
+
+	/** We'll be testing with this existing group */
+	const TESTING_GROUP = Badge_Group_Names::GROUP_GLOBAL_TEMPLATES;
+
+	/** The group we're testing will no longer be considered new from this version onwards  */
+	const VERSION_NO_LONGER_NEW = '16.5';
+	/** The group we're testing will no longer be considered new from this RC version onwards  */
+	const VERSION_NO_LONGER_NEW_RC = '16.5-RC1';
+	/** The group we're testing will still be considered new on this version  */
+	const VERSION_STILL_NEW = '16.4';
+	/** The group we're testing is not considered new on this version  */
+	const VERSION_NEXT_MINOR = '16.6';
+	/** The group we're testing is definitely not considered new on this version  */
+	const VERSION_NEXT_MAJOR = '17.0';
+
```

Comment:
Wouldn't the double asterisk on all the comments above make them docblocks? I guess the should use a single asterisk. Also, there's a double space and missing period at the end.